libcontainer/process_linux.go

```diff
@@ -141,6 +141,16 @@ func (p *setnsProcess) execSetns() error {
 		p.cmd.Wait()
 		return newSystemErrorWithCause(err, "reading pid from init pipe")
 	}
+
+	// Clean up the zombie parent process
+	firstChildProcess, err := os.FindProcess(pid.PidFirstChild)
+	if err != nil {
+		return err
+	}
+	if _, err := firstChildProcess.Wait(); err != nil {
```

Comment:
As discussed IRL, basically this should either just ignore the error:

```
_, _ = firstChildProcess.Wait()
```

Or explicitly special-case `unix.ESRCH` or `unix.ECHILD`, but I'm not convinced that makes much more sense than just ignoring the error (if you `grep` for `Wait` you'll find it).